source/render/MovableTextOverlay.cpp

```diff
@@ -39,7 +40,11 @@ ChildOverlay::ChildOverlay(const Ogre::String& fontName, Ogre::Real charHeight,
     mTimeToDisplay(0),
     mFont(Ogre::FontManager::getSingleton().getByName(fontName))
 {
-    if(!mFont)
+#if defined(OGRE_VERSION) && OGRE_VERSION < 0x10A00
+    if (mFont.isNull())
```

Comment:
Solves build error:
```
/home/travis/build/OpenDungeons/OpenDungeons/source/render/MovableTextOverlay.cpp:42:8: error: no match for ‘operator!’ (operand type is ‘Ogre::FontPtr {aka Ogre::SharedPtr<Ogre::Font>}’)
     if(!mFont)
        ^
```
Not sure if there would be a way to support both versions with the same syntax.

Reply:
@akien-mga  std::shared_ptr-like objects usually have a `.get()` method to obtain the raw pointer:
https://www.ogre3d.org/docs/api/1.9/class_ogre_1_1_shared_ptr.html#a2c3ed4a7b5574c4e53adec450fd537b0

This could be used this way:
```c++
if (mFont.get() == nullptr)
```
WDYT?